scripts/build.ts

```diff
@@ -42,7 +44,7 @@ async function minifyCSS() {
 	for (const id of pluginIds) {
 		const file = path.join(SRC_DIR, `plugins/${id}/prism-${id}.css`);
 		if (fs.existsSync(file)) {
-			input[`plugins/prism-${id}.css`] = file;
+			input[`plugins/${id}/prism-${id}.css`] = file;
```

Comment:
Here we output the `.css` files so they are in the same location in `dist/` as they are in `src/`.

Less confusion: For anything in `src/**/*`, just replace `src/` with `dist/` to arrive at the output path. Easier mental model.